contracts/QuestFactory.sol

```diff
@@ -401,13 +404,24 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         mintFeeRecipient = mintFeeRecipient_;
     }
 
+    /// @dev set a mintFeeRecipient for a specific address
+    /// @param address_ The address of the account
+    /// @param mintFeeRecipient_ The address of the mint fee recipient
+    function setMintFeeRecipientForAddress(address address_, address mintFeeRecipient_) public onlyOwner {
+        if (mintFeeRecipient_ == address(0)) revert AddressZeroNotAllowed();
```

Comment:
If it's set to the zero address that's essentially just re-setting it to default - is it worth preventing this behavior/is this check necessary?